src/Bridges/bridge_optimizer.jl

```diff
@@ -450,6 +495,123 @@ function MOI.set(b::AbstractBridgeOptimizer,
     return MOI.set(b.model, attr, bridged_function(b, value))
 end
 
+# Objective
+
+"""
+    is_objective_bridged(b::AbstractBridgeOptimizer)
+
+Return a `Bool` indicating whether the objective is bridged. The objective is
+said to be bridged if the value of `MOI.ObjectiveFunctionType` is different for
+`b` and `b.model`.
+"""
+is_objective_bridged(b) = !isempty(Objective.bridges(b))
+function _delete_objective_bridges(b)
+    MOI.delete(b, Objective.root_bridge(Objective.bridges(b)))
+    empty!(Objective.bridges(b))
+end
+
+function MOI.supports(b::AbstractBridgeOptimizer,
+                      attr::MOI.ObjectiveFunction{F}) where F
+    if is_bridged(b, F)
+        return supports_bridging_objective_function(b, F)
+    else
+        return MOI.supports(b.model, attr)
+    end
+end
+struct ObjectiveFunctionValue{F<:MOI.AbstractScalarFunction} end
```

Comment:
Why is it necessary to have a different attribute? What's the difference between `ObjectiveFunctionValue` and `ObjectiveValue`?

Reply:
Added a docstring